batcher/batcher.mligo

```diff
@@ -1692,6 +1732,94 @@ let get_valid_swap_reduced
  | Some vswp -> vswp
  | None -> failwith swap_does_not_exist
 
+[@inline]
+let refund_orders
+  (refund_address: address)
+  (ots: ordertypes)
+  (valid_swap:valid_swap)
+  (storage: storage): result =
+  let fee = storage.fee_in_mutez in
+  let collect_refunds ((tam,mutez_to_ref), (ot, amt): ((token_amount_map * tez) * (ordertype * nat))) : (token_amount_map * tez) =
+    let token  = match ot.side with
+                 | Buy -> valid_swap.swap.from.token
+                 | Sell -> valid_swap.swap.to
+    in
+    let ta = {
+       token = token;
+       amount = amt;
+    } in
+    let tam = TokenAmountMap.increase ta tam in
+    let mutez_to_ref = mutez_to_ref + fee in
```

Comment:
Those names are a bit confusing, especially `mutez_to_ref` (as, I believe, references are coming to Cameligo too).

Reply:
mutez_to_refund?

Reply:
For instance, yes.